wiki/Article_styling_criteria/Formatting/de.md

```diff
@@ -0,0 +1,782 @@
+
+```yaml
+no_native_review: true
+```
+
+## Artikelbenennung
+
+*Siehe auch: [Ordnernamen](#ordnernamen) und [Titel](#titel)*
+
+Artikeltitel sollten im Singular stehen und "Sentence case" verwenden. Siehe [Wikipedias Artikel über die Namensgebung](https://de.wikipedia.org/wiki/Wikipedia:Archiv/Richtlinien_f%C3%BCr_die_Namensgebung:Singular-Plural-Debatte) für mehr Details.
+
+Artikeltitel sollten dem Ordnernamen entsprechen, in welchem sie sich befinden (Unterstriche (`_`) dürfen gerne an angemessenen Stellen mit Leerzeichen ersetzt werden). Wenn der Ordnername sich ändert, dann sollte der Titel sich ebenfalls ändern und umgekehrt.
+
+---
+
+Wettbewerbsartikel sind eine Ausnahme. Der Ordner muss Abkürzungen, Akronyme oder Initialien verwenden. Der Titel des Artikels muss den vollen Namen des Wettbewerbs enthalten.
+
+## Ordner- und Dateistrukturen
+
+### Ordnernamen
+
+*Siehe auch: [Artikelbenennung](#artikelbenennung)*
+
+Ordnernamen müssen in Englisch sein und "Sentence case" verwenden.
+
+Ordnernamen dürfen nur diese Zeichen enthalten:
+
+- Groß- und Kleinbuchstaben
+- Zahlen
+- Unterstriche (`_`)
+- Bindestriche (`-`)
+- Ausrufezeichen (`!`)
+
+### Dateinamen für Artikel
+
+Der Dateiname für einen Artikel kann in der Spalte `Dateiname` der Tabelle in [Locales](#locales) gefunden werden. Der übersetzte Artikel muss in denselben Ordner wie der englische Artikel abgelegt werden.
+
+### Indexartikel
+
+Ein Indexartikel muss erstellt werden, wenn der Ordner nur dazu gedacht ist, andere Artikel zu beinhalten. Indexartikel müssen eine Liste an den enthaltenen Artikeln haben. Sie können auch weitere Informationen enthalten, z. B. einen Hauptabsatz oder eine Beschreibung der verlinkten Artikel.
+
+### Artikel für die Begriffsabgrenzung
+
+[Begriffsabgrenzungsartikel](/wiki/Disambiguation) müssen in den Ordner `/wiki/Disambiguation` gesetzt werden. Die Hauptseite muss mit den neuen Artikeln aktualisiert werden. Siehe [Begriffsabgrenzung/Mod](/wiki/Disambiguation/Mod) als ein Beispiel.
+
+Verlinkungen müssen aktualisiert werden, sodass mehrdeutige Begriffe zu den entsprechenden Begriffsabgrenzungsseiten zeigen.
+
+Artikel, die aus einem Begriffsabgrenzungsartikel heraus verlinkt werden, müssen die Kopfnote ["Für andere Bedeutungen"](#für-andere-bedeutungen) tragen.
+
+## HTML
+
+HTML darf nicht verwenden werden, mit der Ausnahme von [Kommentaren](#kommentare). Die Struktur des Artikels muss überarbeitet werden, falls HTML verwendet wurde.
+
+### Kommentare
+
+HTML-Kommentare sollten benutzt werden, um Todos hervorzuheben, aber auch, um Text zu kommentieren. Es wird empfohlen, sie auf eine eigene Zeile zu platzieren, aber es ist auch in Ordnung, wenn sie sich in einem Paragraphen befinden. Wenn das der Fall ist, darf der Start des Kommentars kein Leerzeichen haben.
```

Comment:
```suggestion
HTML-Kommentare sollten benutzt werden, um To-dos hervorzuheben, aber auch, um Text zu kommentieren. Es wird empfohlen, sie auf eine eigene Zeile zu platzieren, aber es ist auch in Ordnung, wenn sie sich in einem Paragraphen befinden. Wenn das der Fall ist, darf der Start des Kommentars kein Leerzeichen haben.
```
[Apparently, this is the correct spelling](https://www.duden.de/rechtschreibung/To_do). However, `Todos` is closer to the HTML comment `TODO`, so I would also be fine with keeping it as is. Same for the next two suggestions.